FizzBuzz.scala

```diff
@@ -0,0 +1,44 @@
+object task3 extends App {
+  def task13(a: String, b: String, c: Int) {
+
+    var word = a
+
+    var word2 = b
```

Comment:
word, word2 are completely unneeded in this method as you already have them as a,b